### DISCUSSION TITLE
Is there a way to use Qutebrowser without typing "i" to input text?

### DISCUSSION THREAD

**Discussion by rhysday** [qutebrowser/qutebrowser#8039] [Q&A] [answered]
I love Qutebrowser, but I've been using it for 2 years and I still can't fully get used to the vim key of typing "i" before text input. The amount of times I've been in the middle of something and gone to type, but ended up typing "d" and closing all my tabs. It also doesn't help that switching tabs takes you out of the text input mode.

**Comment by The-Compiler** [comment, marked as answer]
I took the freedom to convert this to a discussion, as there is nothing actionable for qutebrowser here, and issues should only be used for things that are (bugs and feature requests, mostly).

You can either tweak whatever options that cause qutebrowser to leave insert mode for you (e.g. `tabs.mode_on_change` for the tab switching thing you mentioned); or you could even do without insert mode at all via `input.forward_unbound_keys` and rebinding keys to use a modifier like control.
